src/components/Accordion/Accordion.tsx

```diff
@@ -29,9 +29,9 @@ export interface AccordionProps {
  * Get the minus or plus icon depending on whether the accordion
  * is open or closed.
  */
-const getIcon = (isExpanded = false) => {
+const getIcon = (isExpanded = false, index) => {
   const iconName = isExpanded ? IconNames.Minus : IconNames.Plus;
-  return <Icon name={iconName} size={IconSizes.Small} />;
+  return <Icon id={`${index}-icon`} name={iconName} size={IconSizes.Small} />;
```

Comment:
Change this to
```
id={`accordion-${index}-icon`}
```

to give it a more identifying id.

Reply:
And actually, this may not work if you have more than one `Accordion` component on a page. If you have more than one on the page, the id for the icon will be something like `"0-icon"` or `"accordion-0-icon"`. So you may need to pass in the `id` prop to the `getElementsFromContentData` function to give it a more unique id value. You can try to verify this by testing this

```
const { container } = render(
      <>
        <Accordion
          contentData={[
            {
              label: "Tom Nook",
              panel: (
                <p>
                  Tom Nook, <b>known in Japan as Tanukichi</b>, is a fictional
                  character in the Animal Crossing series who operates the
                  village store.
                </p>
              ),
            },
          ]}
        />
        <Accordion
          contentData={[
            {
              label: "Tom Nook",
              panel: (
                <p>
                  Tom Nook, <b>known in Japan as Tanukichi</b>, is a fictional
                  character in the Animal Crossing series who operates the
                  village store.
                </p>
              ),
            },
          ]}
        />
      </>
    );
    expect(await axe(container)).toHaveNoViolations();
  });
```

Reply:
Made this: `accordian-${id}-icon-${index}`

Reply:
and i am still spelling accordion wrong!